cmd/helm/testdata/helmhome/helm/repository/test-name-index.yaml

```diff
@@ -1,3 +1,3 @@
 apiVersion: v1
 entries: {}
-generated: "2020-06-23T10:01:59.2530763-07:00"
+generated: "2020-07-31T12:50:37.129802-06:00"
```

Comment:
this looks like a test fixture that keeps re-generating any time you run `make test`. While it's unrelated to your PR, we should fix this. Would you mind reverting this for the time being?

Reply:
This got me the other day, too. It's a test fixture that regenerates that file each time.

Reply:
That must be a new bug in that test. I will revert the file in this PR.

Reply:
Fixed. I think I will go hunt down that test fixture error, too. Tests are not supposed to overwrite fixtures.